### DISCUSSION TITLE
2024-01-29 MESH CoP Meeting Notes

### DISCUSSION THREAD

**Discussion by hrynkiwc** [MESH-Model/MESH-Dev#49] [CoP Meetings]
**MESH Community of Practice Meeting Minutes**
_(Prepared by C. Hrynkiw)_

**Location:** Online
	
**Date:** January 29, 2024
	
**Time:** 14:00 CST / 14:00 MDT/ 13:00 YST	

**Duration:**  58 minutes
	
**Attendance:** Mohamed Abdelhamed (SK), Rodolfo Alvarado Montero (U of C), Anthony Bier (YT), Ghislain De Laplante (YT), Babak Farjad (AB), Kamrul Hossain (SK), Charmaine Hrynkiw (ECCC), Kasra Keshavarz (U of C), Alain Pietroniro  (U of C), Dan Princz (ECCC), Frank Seglenieks (ECCC), Prabin Rokaya (AB), Fuad Yassin (ECCC)	

**1.	Update from MESH users and their future plans (Lead: Dan P.):**

_Yukon_ 
- is a supporter of MESH and is using a gridded version 1860 on FEWS.
- U of S is upgrading to a vector-based version, which they expect to validate. Climate change impact application on the Bow River will be transferred to the Yukon River, using higher resolution CanRCM4 forcings. YT would like to support this by acquiring funding through various sources.
- The vector-based version has not been tested yet in FEWS, and it is unknown if FEWS will be able to interpret the structure.

_NHS Transboundary:_
- MESH is being used on the Great Lakes and a setup exists for running past simulations through CaSR (formerly RDRS). Climate change studies are ongoing, with plans to expand to additional Canadian basins.
- Runoff estimates from MESH are being compared with other products.
- While a gridded product would be preferred, a vector model is useful. The vector-based setup is not yet completed for the Great Lakes, although requirements have been met for producing water balance information.
- Climate change (National Adaptation Strategy / Freshwater Security), looking at more basins across Canada

_Saskatchewan:_
- Model developments currently exist for the following basins: Souris, Qu’Appelle, Wood River, Swift Current and the Saskatchewan River Basin.
- An updated version for the Souris is currently in the FEWS system. There are plans to test it in the spring but so far runoff is unlikely due to warm, dry conditions.
- Improvement of the Saskatchewan MESH model is a top priority, and PPWB will be providing a student to do some validation work.
- A second priority is the development of the DZTR component for the Souris. Currently 3 reservoirs are being modelled separately, and it would be preferable to use only one model.
- A third priority is the expansion of models to northern areas, which pose a challenge due to the differing hydrological response in the region.
- Another priority is looking at a vector-based approach.

_Alberta:_
- MESH currently isn’t in use, although it may be used for the Hay River and St. Mary basin. 

_U of Calgary:_
- There is a prairie pothole algorithm in HYPE that is being implemented in MESH and tested in the St. Mary and Milk basins.
- A model-agnostic framework workshop is being planned in March. 
- Thought is being put towards setting up users on a digital alliance cluster and making datasets cloud-based.
- A proposal has been developed to provide a cluster for broader community access. 

**2.	Update from MESH developers (Lead: Dan P.):**
- Meetings are scheduled every two weeks to coordinate code and integrate it into the main code base. The use of GitHub should simplify the process.
- Discussions regarding versions are ongoing.

**3.	Platforms for communications and collaboration (Lead: Dan P.):**
- Recent issues with the implementation of Confluence has resulted in the U of S Wiki being inaccessible to non-U of S users.
- Options for hosting information include GitHub and the GC Wiki.
o	General user information is in the process of being moved over to the GC Wiki, however it will take some time to populate and some things may be missing. Configurations of the model will be on GCWiki, and anyone can make an account: https://wiki.gccollab.ca/MESH
o	Technical and/or developer information will reside on GitHub as well as meeting notes and sharable presentations .

**4.	Terms of Reference Discussion:**
- Users, developers and stewards were discussed.
- Sub-groups will be explored to engage with additional clients.
- The document is still under development.	

**5.	Questions and Discussion:**
- Version 1860 is the most recent.
- There is interest in investigating whether or not MESH can simulate regulated flows in the South Saskatchewan basin, however there are challenges in the speed of simulation for forecasting. While it is possible to capture reservoir operation/management, it is complex and will need to adapt. The DZTR model is used in NSRPS: https://hess.copernicus.org/articles/23/3735/2019/	

**6.	Discussion on Future Topics:**	
- There is interest in investigating DZTR in more depth. Fuad will present information on this at the next meeting.
- MRD coordination (Etienne)
- ChatGPT coding optimization

**7.	Action Items and Person Responsible:**
- None

**8.	Next Meeting:**
- The next MESH CoP meeting is set for Monday, March 11 at 14:00 CST (14:00 MST, 13:00 YST)

**Comment by kasra-keshavarz** [comment]
Thank you so much Charmaine!
